site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -494,6 +494,23 @@ public function showStatus($gradeable_id) {
             $peer = true;
         }
 
+        //get all graded gradeables for queue stats
+        $submissions_in_queue = 0;
+        $gradeables[] = $gradeable;
+        $graded_gradeables = $this->core->getQueries()->getGradedGradeables($gradeables);
+        //check every submission (inactive and active) for every student
+        foreach ($graded_gradeables as $g) {
```

Comment:
I'm worried this nested for loop might be unnecessarily expensive.
Would be better to look files in the queue directory for matches to this assignment.
But I think we can leave it for a future refactor, when eventually 
the queue is stored in the database instead of the file system.